client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/test/index.tsx

```diff
@@ -29,319 +29,204 @@ const messages = {
 		"Looks like your site address is missing its domain extension. Please try again with something like 'example.com' or 'example.net'.",
 };
 
+const continueButton = () => screen.getByRole( 'button', { name: /Continue/ } );
+const siteAddressInput = () => screen.getByLabelText( 'Site address' );
+const usernameInput = () => screen.getByLabelText( 'WordPress admin username' );
+const passwordInput = () => screen.getByLabelText( 'Password' );
+const backupOption = () => screen.getByRole( 'radio', { name: 'Backup file' } );
+const credentialsOption = () => screen.getByRole( 'radio', { name: 'WordPress credentials' } );
+const backupFileInput = () => screen.getByLabelText( 'Backup file location' );
+//TODO: it requires a testid because there is no accessible name, it is an issue with the component
+const specialInstructionsInput = () => screen.getByTestId( 'special-instructions-textarea' );
+const specialInstructionsButton = () =>
+	screen.getByRole( 'button', { name: 'Special instructions' } );
+const skipButton = () =>
+	screen.getByRole( 'button', { name: /Skip, I need help providing access/ } );
```

Comment:
```suggestion
const { getByRole, getByLabelText, getByTestId } = screen;

const continueButton = () => getByRole( 'button', { name: /Continue/ } );
const siteAddressInput = () => getByLabelText( 'Site address' );
const usernameInput = () => getByLabelText( 'WordPress admin username' );
const passwordInput = () => getByLabelText( 'Password' );
const backupOption = () => getByRole( 'radio', { name: 'Backup file' } );
const credentialsOption = () => getByRole( 'radio', { name: 'WordPress credentials' } );
const backupFileInput = () => getByLabelText( 'Backup file location' );
//TODO: it requires a testid because there is no accessible name, it is an issue with the component
const specialInstructionsInput = () => getByTestId( 'special-instructions-textarea' );
const specialInstructionsButton = () => getByRole( 'button', { name: 'Special instructions' } );
const skipButton = () => getByRole( 'button', { name: /Skip, I need help providing access/ } );
```

Reply:
[def978d](https://github.com/Automattic/wp-calypso/pull/94663/commits/def978dc21de2fa0ffef2165ea9a4b1057c8ee28)